src/Particular.CodeRules/Particular.CodeRules.csproj

```diff
@@ -14,7 +13,17 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.2.2" PrivateAssets="All" />
+    <None Remove="AnalyzerReleases.Shipped.md" />
+    <None Remove="AnalyzerReleases.Unshipped.md" />
+  </ItemGroup>
```

Comment:
Not 100% sure if we need this item group. It's what the IDE did when I switched the _Build Action_ for each of these files to _C# analyzer additional file_ as per https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md

Reply:
The default project file rules place any items that don't qualify for other build action ItemGroups into the `None` ItemGroup, so it is correct to remove them from `None` when you add them to to a different one. 

Would this cause a problem if you didn't remove them? Likely not, though if some MSBuild target was enumerating all the ItemGroups, the files would be double counted since they would be in both `None` and `AdditionalFiles`.